kuma/search/serializers.py

```diff
@@ -187,16 +98,16 @@ def get_localized_name(self, obj):
 
 
 class GroupSerializer(serializers.Serializer):
-    name = serializers.CharField(read_only=True)
-    slug = serializers.CharField(read_only=True)
-    order = serializers.CharField(read_only=True)
+    name = serializers.ReadOnlyField()
+    slug = serializers.ReadOnlyField()
+    order = serializers.ReadOnlyField()
 
 
 class FilterWithGroupSerializer(FilterSerializer):
-    tags = serializers.SerializerMethodField('tag_names')
-    group = GroupSerializer(source='group', read_only=True)
+    tags = serializers.SerializerMethodField('get_tag_names')
+    group = GroupSerializer(read_only=True)
 
-    def tag_names(self, obj):
+    def get_tag_names(self, obj):
         return [tag.name for tag in obj.tags.all()]
```

Comment:
This seems like a good optimization spot (not necessarily in this PR) to query just the names and use `values_list` to avoid creating `Tag` model objects just to pluck the name off of them.